### PR TITLE
dev/core#3685 Fix price sets in sample data

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
dev/core#3685 Fix price sets in sample data

https://lab.civicrm.org/dev/core/-/issues/3978

Before
----------------------------------------
Sample price sets for 'Help Support CiviCRM!' contribution page & Summer Solstice event are no longer quick confg but do not show up correctly due to not having financia types

image

After
----------------------------------------
Financial types added

image

Technical Details
----------------------------------------

Comments
----------------------------------------
@christianwach

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@seamuslee001 @demeritcowboy what is required here? When I ran regen locally I didn't get a commitable generated file - but just submitting this tpl change doesn't seem to be enough to flow through

http://core-24926-2a55r.test-3.civicrm.org:8001/civicrm/api4#/explorer/PriceSet/get

**Comment by demeritcowboy** [comment]
Taking a looksee. It might be the args given to regen.

**Comment by demeritcowboy** [comment]
When I run regen.sh I don't pass any arguments. Maybe that does more than it needs to usually but it seems to work here. You can see my generated file at the bottom of the page here if you want to grab that: https://github.com/demeritcowboy/civicrm-core/actions/runs/3422617514

**Comment by demeritcowboy** [comment]
Oh wait that was against master. Don't use that one.

**Comment by demeritcowboy** [comment]
Ok it errors in 5.56 but semi-silently. Let's see why...

File: xml/templates/civicrm_sample.tpl [hunk 1]
```diff
@@ -180,10 +180,10 @@ VALUES
 
 -- Insert sample data for event
 
-INSERT INTO `civicrm_price_set` (`name`,`title`,`is_active`,`extends`, `is_quick_config`)
+INSERT INTO `civicrm_price_set` (`name`,`title`,`is_active`,`extends`, `is_quick_config`, `financial_type_id`)
 VALUES ('rain_forest_cup_youth_soccer_tournament', 'Rain-forest Cup Youth Soccer Tournament', 1, 1, 1),
```
Comment (seamuslee001) [inline xml/templates/civicrm_sample.tpl:184]: won't these 2 rows need NULL or 0 values at the end otherwise column count won't match?

Comment (eileenmcnaughton) [reply xml/templates/civicrm_sample.tpl:184]: ah yeah - it shouldn't be NULL either - should be a value

Comment (eileenmcnaughton) [reply xml/templates/civicrm_sample.tpl]: OK that was it - @demeritcowboy @seamuslee001 I think it's worked now - thanks!

**Comment by eileenmcnaughton** [comment]
@demeritcowboy @seamuslee001 this is working now - if you merge I'll port to 5.55

**Comment by eileenmcnaughton** [comment]
@seamuslee001 @demeritcowboy can we merge this now?

**Comment by eileenmcnaughton** [comment]
thanks @demeritcowboy

**Pull request merged**